python/louis/__init__.py.in

```diff
@@ -49,27 +50,20 @@ def _createTypeformbuf(length, typeform=None):
     """Creates a typeform buffer for liblouis calls"""
     return (c_ushort*length)(*typeform) if typeform else (c_ushort*length)()
 
-createStr = None
+createEncodedByeString = None
 if sys.version_info[0] == 2:
```

Comment:
`sys.version_info.major` is more readable, assuming it works on all current versions of Python.